### ISSUE TITLE
Deprecation warning on install...

### ISSUE THREAD

**Issue by ellisonbg** [takluyver/bash_kernel#40]
Got this when I install against jupyter 4.x:

```
(python34)BrianTempRetinaMBP:.ipython bgranger$ python -m bash_kernel.install
/Users/bgranger/anaconda/envs/python34/lib/python3.5/site-packages/IPython/kernel/__init__.py:13: ShimWarning: The `IPython.kernel` package has been deprecated. You should import from ipykernel or jupyter_client instead.
  "You should import from ipykernel or jupyter_client instead.", ShimWarning)
Installing IPython kernel spec
```

Just needs to be updated to use the new API.

**Comment by takluyver** [comment]
Yep, there's already a PR for it (#39). I'm just leaving it for a while longer in case anyone is still installing it on IPython 3.x.

**Comment by ellisonbg** [comment]
Ahh, got it, feel free to close whenever it makes sense.

On Tue, Oct 20, 2015 at 1:25 PM, Thomas Kluyver <notifications@github.com>
wrote:

> Yep, there's already a PR for it (#39
> <https://github.com/takluyver/bash_kernel/pull/39>). I'm just leaving it
> for a while longer in case anyone is still installing it on IPython 3.x.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/takluyver/bash_kernel/issues/40#issuecomment-149639512>
> .
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by takluyver** [comment]
Closed by #39
